src/components/Pagination/PaginationService.js

```diff
@@ -0,0 +1,87 @@
+// @flow
```

Comment:
I'm not sure that we use `flow` in the Circuit UI, but I might be wrong.

Reply:
It's a leftover :P